sources/packages/api/src/services/education-program-offering/education-program-offering.service.models.ts

```diff
@@ -70,3 +72,14 @@ export interface SaveOfferingModel {
   offeringStatus: OfferingStatus;
   offeringType: OfferingTypes;
 }
+
+/**
+ * Filter object to pass all the offering optional filters.
+ ** offeringTypes defaulted to Public hence it needs to be assigned with value
+ ** only if private offerings are needed from the service.
+ */
+export class OfferingsFilter {
```

Comment:
It is declared as a class but it seems to be used more like an interface. Does it need to be a class?